src/test/java/ru/training/at/hw3/tests/ExerciseFirst.java

```diff
@@ -0,0 +1,128 @@
+package ru.training.at.hw3.tests;
+
+import org.openqa.selenium.support.PageFactory;
+import org.testng.annotations.Test;
+import org.testng.asserts.SoftAssert;
+import ru.training.at.hw3.pages.Header;
+import ru.training.at.hw3.pages.HomePage;
+import ru.training.at.hw3.utils.WaitActions;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.util.Properties;
+
+
+public class ExerciseFirst extends TestBase {
+    SoftAssert softAssert = new SoftAssert();
+
+    @Test
+    public void homePageTestWithSoftAsserts() {
+        File data = new File("src/test/resources/hw3properties/" +
+                "test.properties");
+        FileInputStream fileInput = null;
+        try {
+            fileInput = new FileInputStream(data);
+        } catch (FileNotFoundException e) {
+            e.printStackTrace();
+        }
+        Properties prop = new Properties();
+        try {
+            prop.load(fileInput);
+        } catch (IOException e) {
+            e.printStackTrace();
+        }
```

Comment:
Why do you read data from the file properties here? Looks like code duplication from **src/test/java/ru/training/at/hw3/pages/HomePage.java**